DataStructures/Dictionaries/OpenAddressingHashTable.cs

```diff
@@ -0,0 +1,225 @@
+﻿/***
+ * Open Addressing Hash Table
+ * 
+ * Implementation of Open Addressing Hash Table using
+ *      - Linear Probing (coming soon)
+ *      - Quadratic Probing (coming soon)
+ *      - Double Hashing
+ *      
+ * Author: Samuel Kenney
+ * Created: 6/7/2017
+ * Last Modified: 6/13/2017
+ * 
+ */
```

Comment:
Please remove the author block comment. Contributions are automatically added on the GitHub Repo's [Contributors section|https://github.com/aalhour/C-Sharp-Algorithms/graphs/contributors], and we maintain a list of contributors on the [REAMDE's Contributors section](https://github.com/aalhour/C-Sharp-Algorithms#contributors) as well.